### PR TITLE
Show how to enable all origins in HTTP CORS section

### PR DESCRIPTION
**Pull request by sberyozkin**
FYI, enabling a wildcard automatically in devmode would not be a good idea IMHO, as one would not know until the application starts in prod that the CORS is no longer supported unless the origins are configured.

Focusing the developers' attention on the `%dev` profile when enabling the wildcard will help them avoid any last minute surprises in prod

### CODE REVIEW

File: docs/src/main/asciidoc/http-reference.adoc [hunk 1]
```diff
@@ -328,6 +328,20 @@ quarkus.http.cors.access-control-max-age=24H
...
+
+Having to configure required origins when you only start developing a Quarkus application requiring CORS support can be difficult and in such cases you may want to enable all origins in devmode only in order to focus on the actual application development first:
+
+[source, properties]
+----
+quarkus.http.cors=true
+%dev.quarkus.http.cors.origins=/.*/
+----
+
+It is important that you enable all origins only in dev profile, supporting all origins in production is not recommended if you would like CORS filter be effective in blocking unauthorized origins.
```
Comment (gsmet) [inline docs/src/main/asciidoc/http-reference.adoc:343]:
```suggestion
It is important that you enable all origins only for the dev profile, allowing all origins in production is not recommended and could expose your applications to serious security issues.
```

File: docs/src/main/asciidoc/http-reference.adoc [hunk 2]
```diff
@@ -328,6 +328,20 @@ quarkus.http.cors.access-control-max-age=24H
 quarkus.http.cors.access-control-allow-credentials=true
 ----
 
+`/https://([a-z0-9\\-_]+)\\.app\\.mydomain\\.com/` is treated as a regular expression because it is surrounded by forward slash characters.
+
+=== Support all origins in devmode
+
+Having to configure required origins when you only start developing a Quarkus application requiring CORS support can be difficult and in such cases you may want to enable all origins in devmode only in order to focus on the actual application development first:
```
Comment (gsmet) [inline docs/src/main/asciidoc/http-reference.adoc:335]:
```suggestion
Having to configure required origins when you start developing a Quarkus application requiring CORS support can be difficult and, in such cases, you may want to allow all origins in dev mode only in order to focus on the actual development first:
```

**Review by gsmet — DISMISSED** [review]
I suggested some small improvements.

**Comment by sberyozkin** [comment]
Thanks @gsmet , let me squash now

**Review by ebullient — APPROVED** [review]
Thank you.

**Pull request merged**